src/pages/[platform]/build-a-backend/add-aws-services/analytics/storing-data/index.mdx

```diff
@@ -0,0 +1,188 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Storing analytics data',
+  description: 'The Amazon Data Firehose analytics provider allows you to send analytics data to an Amazon Data Firehose stream for reliably storing data.',
+  platforms: [
+    'javascript',
+    'angular',
+    'nextjs',
+    'react',
+    'vue',
+    'react-native',
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+The Amazon Data Firehose analytics provider allows you to send analytics data to an [Amazon Data Firehose](https://aws.amazon.com/firehose/) stream for reliably storing data.
+
+## Setup Firehose stream
+
+The following is an example utilizing the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html) to create the Analytics resource powered by [Amazon Data Firehose](https://aws.amazon.com/firehose/).
+
+```ts title="amplify/backend.ts"
+import { Bucket } from "aws-cdk-lib/aws-s3";
+import { CfnDeliveryStream } from "aws-cdk-lib/aws-kinesisfirehose";
+import { Stack , RemovalPolicy} from "aws-cdk-lib";
+import {
+  Policy,
+  PolicyStatement,
+  Role,
+  ServicePrincipal,
+} from "aws-cdk-lib/aws-iam";
+
+const backend = defineBackend({
+  auth, 
+  data,
+  // additional resources 
+});
+
+// Create a new stack for the Firehose resources
+const firehoseStack = backend.createStack("firehose-stack");
+
+// Create a new S3 bucket for the Firehose destination
+const s3Bucket = new Bucket(firehoseStack, "FirehoseDestinationBucket", {
+  // removalPolicy: cdk.RemovalPolicy.DESTROY, // Use with caution in production environments
+  // autoDeleteObjects: true, // Use with caution in production environments
+});
+
+// Create a new IAM role for the Firehose
```

Comment:
any reason, we don't re use the `defineStorage` bucket?

Reply:
switched to use defineStorage